src/components/NavBar.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from "react";
+import { getTopics } from "../utils/api-calls";
+
+
+const NavBar = ()=> {
+
+    const [topics, setTopics] = useState([]);
+    
+
+    useEffect(()=> {
+        setIsLoading(true);
+        getTopics()
```

Comment:
I know this is for later, but we don't have any requirement to get the topics for this ticket